geode-core/src/main/java/org/apache/geode/cache/query/internal/DefaultQueryService.java

```diff
@@ -104,10 +104,21 @@
    * implementation of MethodInvocationAuthorizer, which provides the same functionality.
    */
   @Deprecated
-  public static final boolean ALLOW_UNTRUSTED_METHOD_INVOCATION = Boolean.getBoolean(
-      GEMFIRE_PREFIX + "QueryService.allowUntrustedMethodInvocation");
-
-  private boolean deprecatedWarningHasBeenShown = false;
+  public static final boolean ALLOW_UNTRUSTED_METHOD_INVOCATION;
+
+  public static final String DEPRECIATION_WARNING = "The property " + GEMFIRE_PREFIX +
```

Comment:
Small correction: the word should read `DEPRECATION` instead of `DEPRECIATION`.

Reply:
Done